### PR TITLE
add additional data tags to summary hook html

### PR DESCRIPTION
**Pull request by ashultz0**
This allows the ai-spot service to do record keeping about summary quality.

**Merge checklist:**
Check off if complete *or* not applicable:
- [x] Version bumped
- [x] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Fixup commits are squashed away
- [x] Unit tests added/updated
- [x] Manual testing instructions provided
- [x] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: edx_arch_experiments/summaryhook_aside/block.py [hunk 1]
```diff
@@ -8,11 +8,14 @@
 from edx_arch_experiments.summaryhook_aside.waffle import summary_enabled, summary_staff_only
 
 summary_fragment = """
+<div>&nbsp;</div>
```
Comment (rgraber) [inline edx_arch_experiments/summaryhook_aside/block.py:11]: is this supposed to be here?

**Review by rgraber — COMMENTED** [review]
One question

File: edx_arch_experiments/summaryhook_aside/block.py (hunk 1 above)
Comment (ashultz0) [reply edx_arch_experiments/summaryhook_aside/block.py:11]: Good eye but actually yes! it's a spacer to keep the result box from rubbing against the staff buttons which annoys the UX guy, the first diff puts this in on purpose.

**Review by rgraber — APPROVED** [review]
LGTM

**Pull request merged**